README.md

```diff
@@ -85,6 +86,8 @@ Property | Required | Description
 JAEGER_SERVICE_NAME | yes | The service name
 JAEGER_AGENT_HOST | no | The hostname for communicating with agent via UDP
 JAEGER_AGENT_PORT | no | The port for communicating with agent via UDP
+JAEGER_GRPC_TARGET | no | The target for communicating with collector via GRPC
+JAEGER_GRPC_ROOT_CERTIFICATE | no | The root certificate used to create a trusted TLS connection to the GRPC collector
```

Comment:
Could use some explanation - according to [docs][1], this is not using a client certificate pair, but only the CA certs.

[1]: https://grpc.github.io/grpc/csharp/api/Grpc.Core.SslCredentials.html#Grpc_Core_SslCredentials__ctor_System_String_

Reply:
Thanks for pointing out. Misremembered it when documenting. This is the PEM content of the root certificate and not the path. Do you prefer the path or PEM content?

Reply:
Don't have a preference. File sounds a bit simpler to manage, and that's what server options support.

Reply:
Fixed it to use the path .This also allows us to use secrets in docker.